packages/cli/src/commands/process/create.ts

```diff
@@ -36,6 +37,23 @@ export default class ProcessCreate extends Command {
     if (hashes.length != 1) throw new Error('error while getting the hash of the process created')
     const hash = hashes[0]
     this.spinner.stop(hash)
-    return this.lcd.process.get(hash)
+    const process = await this.lcd.process.get(hash)
+    this.createTransfer(process, mnemonic)
+    return process
+  }
+
+  async createTransfer(process: IProcess, mnemonic: string) {
```

Comment:
is this still necessary? it's done by the process keeper already: https://github.com/mesg-foundation/engine/pull/1699/files#diff-7e9b501b837ce29ef68a80f1cfc21534

Reply:
removed